test/distributions/vass/testReplicatedDist1.chpl

```diff
@@ -124,6 +139,13 @@ on teston {
   showA();
   resetA();
 
+  start("A[DsubLoc] = ARepl;");
+  reset();
+  resetA();
+  A[DsubLoc] = ARepl;
+  showA();
+  resetA();
+
   // parallel loop
   start("forall (b,a) in zip(A[Dsub],ARepl) do a = b;");
   reset();
```

Comment:
How about uncomment the two snippets below and see how they work?

Also, uncommented or not, would you check whether it should be `Dsub` or `DsubLoc` ?